rules/REQUEST-920-PROTOCOL-ENFORCEMENT.conf

```diff
@@ -433,7 +433,7 @@ SecRule REQUEST_URI "@rx \%(?:(?!$|\W)|[0-9a-fA-F]{2}|u[0-9a-fA-F]{4})" \
 
 # Not supported by re2 (?!re).
 #
-SecRule REQUEST_HEADERS:Content-Type "@rx ^(?:application\/x-www-form-urlencoded|text\/xml)(?:;(?:\s?charset\s?=\s?[\w\d\-]{1,18})?)??$" \
+SecRule REQUEST_HEADERS:Content-Type "@rx ^application\/x-www-form-urlencoded(?:;(?:\s?charset\s?=\s?[\w\d\-]{1,18})?)??$" \
```

Comment:
Since we are modifying this line, I think we should drop the (?:...) and make the regexp case insensitive since the type and subtypes are not case sensitive, e.g.:

"@rx ^(?i)application/x-www-form-urlencoded"

Is IMO better.